lib/modules/noParticipation.js

```diff
@@ -108,8 +108,8 @@ function isSubscriber() {
 	return document.body.classList.contains('subscriber');
 }
 
-function nonNpLocation() {
-	const nonNpUrl = new URL(location.href);
+export function nonNpLocation(path: ?string) {
```

Comment:
This can be `path: string | null`, which is slightly stronger.

Reply:
Thank you! I'll make the change